pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -890,3 +891,90 @@ func TestVolumeBinding(t *testing.T) {
 		})
 	}
 }
+
+func TestIsSchedulableAfterCSIStorageCapacityChange(t *testing.T) {
+	table := []struct {
+		name   string
+		pod    *v1.Pod
+		oldCap interface{}
+		newCap interface{}
+		err    bool
+		expect framework.QueueingHint
```

Comment:
nit: 

```suggestion
		expectedErr  bool
		expectedHint framework.QueueingHint
```

Reply:
As the only return value other than QHint is err, I think using `expect` would be sufficient without explicitly using `expectedHint`.

Reply:
Personally, I prefer the field names for test case inputs and expected results to be easily distinguishable. But that's just a matter of preference, and I don't have a strong opinion about it. Even though I don't have permission to do so, that's why I `/lgtm`'d it.